pkg/plugins/golang/v4/scaffolds/internal/templates/test/e2e/test.go

```diff
@@ -44,106 +44,101 @@ var TestTemplate = `{{ .Boilerplate }}
 package e2e
 
 import (
+	"context"
 	"fmt"
+	"github.com/go-resty/resty/v2"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"net/http"
 	"os/exec"
+	"strconv"
 	"time"
 
-    . "github.com/onsi/ginkgo/v2"
-    . "github.com/onsi/gomega"
-	
 	"{{ .Repo }}/test/utils"
 )
 
 const namespace = "{{ .ProjectName }}-system"
+const deploymentName = "{{ .ProjectName }}-controller-manager"
 
 var _ = Describe("controller", Ordered, func() {
 	BeforeAll(func() {
-		By("installing prometheus operator")
-		Expect(utils.InstallPrometheusOperator()).To(Succeed())
+		var err error
+		By("prepare kind environment", func() {
+			cmd := exec.Command("make", "kind-prepare")
```

Comment:
Same we cannot add anything that specific for a vendor.

Reply:
Let's discuss vendor(kind)-)-related issue in a certain thread, to not spread this discussion over all threads, I'd suggest the [first one](https://github.com/kubernetes-sigs/kubebuilder/pull/3989#discussion_r1649181013).